.rubocop.yml

```diff
@@ -20,6 +20,32 @@ Naming/PredicateName:
     - def_node_matcher
     - def_node_search
 
+Naming/UncommunicativeBlockParamName:
+  AllowedNames:
+    - 'child_node1'
+    - 'child_node2'
+    - 'expression1'
+    - 'expression2'
+    - 'node1'
+    - 'node2'
+    - 'operator1'
+    - 'operator2'
+    - 'range1'
+    - 'range2'
+    - 'token1'
+    - 'token2'
+    - 'var1'
+    - 'var2'
```

Comment:
I find this list informative. Seems like these cops could also use a config option to turn disable `NamesEndingInNumbers`.

Reply:
I was just about to suggest this. :-)

Reply:
I'll add this and disable it by default as well.